src/controllers/certificates/certificate_secret.go

```diff
@@ -34,7 +41,10 @@ func (certSecret *certificateSecret) setSecretFromReader(ctx context.Context, ap
 
 	switch {
 	case k8serrors.IsNotFound(err):
-		certSecret.secret = kubeobjects.NewSecret(buildSecretName(), namespace, map[string][]byte{})
+		certSecret.secret, err = kubeobjects.CreateSecret(certSecret.scheme, certSecret.owner, kubeobjects.NewSecretNameModifier(buildSecretName()), kubeobjects.NewSecretNamespaceModifier(namespace), kubeobjects.NewSecretDataModifier(map[string][]byte{}))
```

Comment:
use multiple lines, like in https://github.com/Dynatrace/dynatrace-operator/pull/1506/files#diff-6610f9b2413c0f2595d0ba1d2aa088afff368fce6d0792a728c92d123d912c48R62-R67